client/src/components/ListMember.js

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components';
+import { Pen } from '../icons/Pen';
+import { Trashbin } from '../icons/Trashbin';
+
+export default function ListMember({ member }) {
+  return (
+    <Member>
+      {member.firstName} {member.lastName} <PenStyled /> <TrashbinStyled />
+    </Member>
+  );
+}
+
+const Member = styled.div`
```

Comment:
A `section` element would be nicer. However, since your components name is `ListMember` I initially thought this would be a `li` element wrapped in a `ul` or `ol`. Maybe consider changing the name or the elements.
```suggestion
const Member = styled.section`
```